PhpAmqpLib/Channel/AbstractChannel.php

```diff
@@ -218,7 +218,7 @@ public function next_frame($timeout = 0)
     }
 
     /**
-     * @param $method_sig
+     * @param string $method_sig
      * @param string $args
```

Comment:
While you're at it, looks like `$args` is most commonly of type `AMQPWriter`.  Maybe change this to `AMQPWriter|string`